### PR TITLE
Fix/6104/backup community key field

### PR DESCRIPTION
**Pull request by osmaczko**
fixes: https://github.com/status-im/status-desktop/issues/6104

### CODE REVIEW

File: ui/app/AppLayouts/Chat/popups/community/TransferOwnershipPopup.qml [hunk 1]
```diff
@@ -1,87 +1,75 @@
...
-                        popup.store.copyToClipboard(popup.privateKey)
-                    }
+        spacing: 16
+
+        StatusInput {
+            id: pKeyInput
+
+            Layout.fillWidth: true
+
+            property string elidedPkey: root.privateKey.substring(0, 15) + "..." + root.privateKey.substring(root.privateKey.length - 16)
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/popups/community/TransferOwnershipPopup.qml:33]: `readonly`

**Comment by caybro** [comment]
Otherwise looks nice

**Comment by osmaczko** [comment]
> Otherwise looks nice

@caybro Thanks, could you please approve?

**Review by caybro — APPROVED** [review]

File: ui/app/AppLayouts/Chat/popups/community/TransferOwnershipPopup.qml [hunk 2]
```diff
@@ -1,87 +1,75 @@
...
-                        popup.store.copyToClipboard(popup.privateKey)
-                    }
+        spacing: 16
+
+        StatusInput {
+            id: pKeyInput
+
+            Layout.fillWidth: true
+
+            readonly property string elidedPkey: root.privateKey.substring(0, 15) + "..." + root.privateKey.substring(root.privateKey.length - 16)
```
Comment (endulab) [inline ui/app/AppLayouts/Chat/popups/community/TransferOwnershipPopup.qml:33]: In Utils.qml there is a funtion getElidedCompressedPk with very same funtionality. Can you reuse it ?

**Review by endulab — CHANGES_REQUESTED** [review]

**Review by Hbouaz — APPROVED** [review]
Passed testing

**Review by endulab — APPROVED** [review]

**Pull request merged**